server/opendp_apps/dataset/models.py

```diff
@@ -130,13 +101,13 @@ class DataverseFileInfo(DataSetInfo):
     """
     Refers to a DV file from within a DV dataset
     """
-    # TODO: This should have all fields from DV API response
     dv_installation = models.ForeignKey(RegisteredDataverse, on_delete=models.PROTECT)
     dataverse_file_id = models.IntegerField()
     dataset_doi = models.CharField(max_length=255)
     file_doi = models.CharField(max_length=255, blank=True)
     dataset_schema_info = models.JSONField(null=True, blank=True)
     file_schema_info = models.JSONField(null=True, blank=True)
+    depositor_setup_info = models.OneToOneField('analysis.DepositorSetupInfo', on_delete=models.PROTECT, null=True)
```

Comment:
This should be added to the `DataSetInfo` object. The steps will be similar for an UploadedFile

Reply:
When I tried the dataset-info endpoint, I didn't see the depositor_setup_info object_id being returned, so I think that needs to be added.  
Also, the current file name being in dataset-info returned isn't very descriptive - in our MockDV example: "doi:10.7910/DVN/PUXVDH (Mock Local Dataverse (http://127.0.0.1:8000/dv-mock-api))".  There is a more descriptive name in dataset_schema_info.name: name: "Replication Data for: Eye-typing experiment"

Reply:
Which name are you referring to here? When I get http://localhost:8000/api/dataset-info/ I receive this response:

`{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "object_id": "9255c067-e435-43bd-8af1-33a6987ffc9b",
            "name": "Fatigue_data.tab",
            "created": "2021-03-23T17:22:50.889000Z",
            "creator": "dev_admin",
            "installation_name": "Harvard Dataverse",
            "dataverse_file_id": 4164587,
            "dataset_doi": "doi:10.7910/DVN/PUXVDH",
            "file_doi": "",
            "status": "step_100",
            "depositor_setup_info": {
                "id": 1,
                "created": "2021-03-23T17:22:50.889000Z",
                "updated": "2021-03-30T20:39:14.177000Z",
                "object_id": "9255c067-e435-43bd-8af1-33a6987ffc9b",
                "is_complete": false,
                "user_step": "step_100",
                "epsilon": null,
                "dataset_questions": null,
                "variable_ranges": null,
                "variable_categories": null
            },
            "resourcetype": "DataverseFileInfo"
        }
    ]
}`

Reply:
I'm referring to the data returned from the dataverse, that's used to create a DataverseFileInfo object.  It's the property dataset_schema_info.name in the json returned from the Dataverse.  So the change would be in the dv-file endpoint.  If we change the name assigned in the dv-file create, it will show up in the name here.  If this is too unrelated to the ticket, let me know, it just occurred to me because we're using the name in the My Data dataset list.

Reply:
Sure, let's put that under a separate ticket

Reply:
ok, sounds good :)